gcloud-java-storage/src/main/java/com/google/gcloud/spi/StorageRpc.java

```diff
@@ -42,6 +42,8 @@
     IF_SOURCE_METAGENERATION_NOT_MATCH("ifSourceMetagenerationNotMatch"),
     IF_SOURCE_GENERATION_MATCH("ifSourceGenerationMatch"),
     IF_SOURCE_GENERATION_NOT_MATCH("ifSourceGenerationNotMatch"),
+    IF_MD5_MATCH("md5Hash"),
```

Comment:
Actually, I am not sure about my naming suggestion now though I still hope we may not need them..